src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form"
+      data-qa="big"
+      action="http://localhost:8080/"
+      method="post"
+    >
+      <label class="label">
+        <input
+        class="search"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+      </label>
+    </form>
+
+    <form
+      class="form"
+      data-qa="small"
+      action="http://localhost:8080/"
+      method="post"
+    >
+      <label class="label label-small">
+        <input
+        class="search search-small"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
```

Comment:
```suggestion
        <input
          class="search search-small"
          type="text"
          data-qa="keypress"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```
Consider fixing indents according to the code style guide